README.md

```diff
@@ -271,6 +271,36 @@ Add to your repository as `.github/workflows/sile.yaml`.
 This work flow assumes your project has a source file `my-document.sil` and will leave behind a `my-document.pdf`.
 Note the comments in [the section about Docker](#docker) regarding version tags.
 
+## Installing third-party packages
+
+Third-party SILE packages rely on the `luarocks` package manager.
+
+To install the latest version, for the given package, you may often use its provided “rockspec”:
+
+```bash
+# This example uses markdown.sile, a highly useful plugin that converts Markdown documents to SILE `.sil`.
```

Comment:
Not that I mind ;) but it doesn't converts to .sil (yet...). It allows reading and processing Markdown documents with SILE

Reply:
I reworded this with that in mind.